### PR TITLE
BRING BACK HANK

### PR DESCRIPTION
**Pull request by cjdenio**
BRING BACK HANK

### CODE REVIEW

**Comment by cjdenio** [comment]
please ignore the fact that i made ui.js an erb file ([this is strange but valid](https://guides.rubyonrails.org/asset_pipeline.html#css-and-erb))

File: app/assets/javascripts/ui.js.erb [hunk 1]
```diff
@@ -402,3 +402,15 @@ document.addEventListener("turbo:load", () => {
     document.body.classList.remove('embedded');
   }
 })
+
+let hankIndex = 0
+$(document).on('keydown', function (e) {
+  if (e.originalEvent.key === 'hank'[hankIndex]) {
+    hankIndex++
+    if (hankIndex === 4) {
+      return $('[name="header-logo"]').attr('src', "<%= asset_path 'hank.png' %>")
```
Comment (garyhtou) [inline app/assets/javascripts/ui.js.erb]: Are we able to place `hank.png` in `/public` and link to it as `/hank.png`?

Comment (cjdenio) [reply app/assets/javascripts/ui.js.erb]: i suppose that is more reasonable

File: .vscode/settings.json [hunk 2]
```diff
@@ -29,5 +29,8 @@
   "[ruby][gemfile][erb]": {
     "editor.tabSize": 2,
     "editor.insertSpaces": true
+  },
+  "files.associations": {
+    "*.js.erb": "javascript"
```
Comment (garyhtou) [inline .vscode/settings.json]: i suppose that might not be needed anymore

**Review by garyhtou — APPROVED** [review]
let's bring back hank!!!

(i never knew hank existed)

**Review by sarthaktexas — APPROVED** [review]

**Review by sampoder — APPROVED** [review]

**Pull request merged**